operator-framework/src/main/java/io/javaoperatorsdk/operator/ControllerUtils.java

```diff
@@ -1,89 +1,39 @@
 package io.javaoperatorsdk.operator;
 
+import io.fabric8.kubernetes.client.CustomResource;
 import io.javaoperatorsdk.operator.api.Controller;
 import io.javaoperatorsdk.operator.api.ResourceController;
-import io.fabric8.kubernetes.api.builder.Function;
-import io.fabric8.kubernetes.client.CustomResource;
-import io.fabric8.kubernetes.client.CustomResourceDoneable;
-import javassist.*;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.HashMap;
-import java.util.Map;
 
 
 public class ControllerUtils {
-
-    private final static double JAVA_VERSION = Double.parseDouble(System.getProperty("java.specification.version"));
-    private static final String FINALIZER_NAME_SUFFIX = "/finalizer";
-
-    // this is just to support testing, this way we don't try to create class multiple times in memory with same name.
-    // note that other solution is to add a random string to doneable class name
-    private static Map<Class<? extends CustomResource>, Class<? extends CustomResourceDoneable<? extends CustomResource>>>
-            doneableClassCache = new HashMap<>();
-
+    private static final String FINALIZER_NAME_SUFFIX = ".javaoperatorsdk.io/finalizer";
```

Comment:
Now it should be fine to revert this part, right?

Reply:
:+1: